documentation/tutorials/StateAssimilation/TreeRingSDA.Rmd

```diff
@@ -80,7 +80,7 @@ Perform a site-level SIPNET run using the following settings
   </spin.up>
   <forecast.time.step>1</forecast.time.step>
   <start.date>1980/01/01</start.date>
-  <end.date>2014/12/31</end.date>
+  <end.date>2016/12/31</end.date>
```

Comment:
I've always had to use https to pull from github on VM. Probably some port mapping thing. 

Ankur Desai, UW-Madison
 +1-608-218-4208 [mobile]

> On Aug 17, 2016, at 10:07 PM, istfer <notifications@github.com> wrote:
> 
> In documentation/tutorials/StateAssimilation/TreeRingSDA.Rmd:
> 
> > @@ -80,7 +80,7 @@ Perform a site-level SIPNET run using the following settings
> >    </spin.up>
> >    <forecast.time.step>1</forecast.time.step>
> >    <start.date>1980/01/01</start.date>
> > -  <end.date>2014/12/31</end.date>
> > +  <end.date>2016/12/31</end.date>
> Maybe you can leave it as it is and if others have error with the ssh address as I did on the VM we can tell them to use the https, as you and @mdietze wish
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>